### DISCUSSION TITLE
m-serial-bin.h , line 102, missing =

### DISCUSSION THREAD

**Discussion by Cilyena** [P-p-H-d/mlib#103] [General]
Hello,

`int l = (c = 255) ? 8 : (c == 254) ? 4 : 2;`
to
`int l = (c == 255) ? 8 : (c == 254) ? 4 : 2;`

**Comment by P-p-H-d** [comment]
Hello,
This is indeed an obvious error, and I don't understand why the static tool analyzers didn't catch it!
Thanks for your finding!
It has been fixed in master.
